src/labels/CustomLabels.labels

```diff
@@ -2028,7 +2028,7 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>healthSolutionAccountModel</shortDescription>
-        <value>All of your individual Contacts should use the same Account Model. View these Contacts by running the &lt;b&gt;Individual Contacts by Account Type&lt;/b&gt; report in the NPSP Health Check reports folder.</value>
+        <value>All of your Accounts should use the same Account Model. View these Accounts by running the &lt;b&gt;Individual Accounts by Account Type&lt;/b&gt; report in the NPSP Health Check reports folder.</value>
```

Comment:
@jvarese The solution message text for the Account Model Data health check has been updated to refer to the new report.

<img width="1007" alt="screen shot 2016-07-18 at 8 12 44 pm" src="https://cloud.githubusercontent.com/assets/1058323/16949811/5f6b54ec-4d82-11e6-914f-7a50ba8e4d23.png">